frontend/src/Onboarding/LandingScreen.tsx

```diff
@@ -129,6 +133,11 @@ export class LandingScreen extends React.Component<Props> {
     this.dev = process.env.NODE_ENV === "development";
   }
 
+  componentWillMount() {
+    // make an API request to searchNEU to get the supported majors and their corresponding plans.
+    this.props.fetchMajorsAndPlans();
```

Comment:
fetching major here and check if majors length is 0 in the onboarding screen component